imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/ExoPlayerWithAdPlayback.java

```diff
@@ -494,10 +494,10 @@ public void onShuffleModeEnabledChanged(boolean shuffleModeEnabled) {
     }
 
     @Override
-    public void onPlayerError(ExoPlaybackException error) {
-        log.d("onPlayerError error = " + error.getMessage());
+    public void onPlayerError(PlaybackException playbackException) {
+        log.d("onPlayerError error = " + playbackException.getMessage());
         stopUpdatingAdProgress();
-        sendSourceError(error);
+        sendSourceError(playbackException);
```

Comment:
Check the error flow.

Reply:
This is also fine. We are getting the proper type of error and sending it to the app with the exception in AdEvent.Error